### DISCUSSION TITLE
stopping container tabs in Firefox

### DISCUSSION THREAD

**Discussion by cmcjsrb** [Cookie-AutoDelete/Cookie-AutoDelete#1291] [Q&A / Support]
I do not use container tabs, for anything in Firefox--even in v. 95.02. Even with the check mark out of the option in the menu of Cookie auto delete to enable container tabs, (which it should not do) with no check mark to enable it, Firefox refuses to stop it.  It will, if you look in about:config, the line that pertains to it running, still be set to true.  How do you disable container tabs in cookie auto delete if taking the check mark out of that option does not do it.

**Comment by kennethtran93** [comment]
Per https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/contextualIdentities, any extension that has the 'contextualidentites' permission automatically turns on containers in Firefox.  We have no control over that as this permission cannot be made optional.  The setting in CAD only affects whether we perform cleanups in containers.

To disable containers in Firefox, you would need to manually modify/override the value you see in `about:config` for `privacy.userContext.enabled` to `false` by double-clicking on it.

**Reply by cmcjsrb** [reply]
I have done that--with a suggestion in FF--with the suggestion to someone else that I read,,did not work for them and does the same for me, it does not work.  the only way they got it to work was disable the extension.  I have tried that too, and that is the only way the setting in about;config will stay false, is by disabling the cookie auto delete extension.  other wise it just goes back to true.

**Reply by kennethtran93** [reply]
Then this is probably something that you'll need to bring up to the Firefox team instead.

As it stands, whether or not it is enabled shouldn't affect your existing workflow.  If you don't use containers, having it enabled won't affect how cleanups are done.

**Comment by kennethtran93** [comment]
For the purpose of this discussion, I have created a bug report for the firefox team to look into here: https://bugzilla.mozilla.org/show_bug.cgi?id=1748576
